Configuration/Geometry/python/GeometryExtended2023RecoReco_cff.py

```diff
@@ -0,0 +1,52 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Hi @ebrondol , @ianna , this file (GeometryExtended2023RecoReco_cff.py) is indeed not needed, probably a remant of the branch when we were preparing the PR and debating the geometry name to be used - but indeed not needed anymore